### PR TITLE
Issue 4541 flask environ builder

### LINKED ISSUE #4541: flask.testing.EnvironBuilder missing
`make_test_environ_builder` is deprecated and we're supposed to use `EnvironBuilder` directly but it's missing from the typestub.

### PR DESCRIPTION
**Pull request by mitch292**
Resolves #4541 

Move from `make_test_environ_builder` to `EnvironBuilder`

### CODE REVIEW

File: third_party/2and3/flask/testing.pyi [hunk 1]
```diff
@@ -6,17 +6,7 @@ from typing import IO, Any, Iterable, Mapping, Optional, Text, TypeVar, Union
...
-
-def make_test_environ_builder(
-    app: Any,
-    path: str = ...,
-    base_url: Optional[Any] = ...,
-    subdomain: Optional[Any] = ...,
-    url_scheme: Optional[Any] = ...,
-    *args: Any,
-    **kwargs: Any,
-): ...
```
Comment (srittau) [inline third_party/2and3/flask/testing.pyi:19]: While `make_test_environ_builder()` has been deprecated in the last released version, it still exists, so we should keep it for now.

File: third_party/2and3/flask/testing.pyi [hunk 2]
```diff
@@ -44,3 +34,17 @@ class FlaskCliRunner(CliRunner):
         color: bool = ...,
         **extra: Any,
     ) -> Result: ...
+
+class EnvironBuilder(WerkzeugEnvironBuilder):
+    app: Any = ...
```
Comment (srittau) [inline third_party/2and3/flask/testing.pyi:39]: Nit: The ` = ...` part is superfluous.

**Review by srittau — CHANGES_REQUESTED** [review]
Thanks, one issue (and one nit) below.

**Comment by mitch292** [comment]
@srittau - Made those changes!

**Review by srittau — APPROVED** [review]

**Pull request merged**